CMakeLists.txt

```diff
@@ -263,8 +263,9 @@ if(WITH_OPENMP)
   find_package(OpenMP COMPONENTS C CXX)
 endif()
 if(OpenMP_FOUND)
-  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
+   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
```

Comment:
Why is this in the diff github? o.O I see no difference. Whitespace?

Reply:
Probably, I'll look at it.